src/include/migraphx/matcher.hpp

```diff
@@ -844,6 +857,12 @@ auto skip_broadcasts_converts(Ms... ms)
     return skip(name("broadcast", "multibroadcast", "contiguous", "convert"))(ms...);
 }
 
+template <class... Ms>
+auto skip_broadcasts_transposes(Ms... ms)
```

Comment:
add contiguous to name as well . Rename to
`skip_broadcasts_transposes_contiguous`

Reply:
Done